client/app/components/conference_room/ConferenceRoomContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import moment from 'moment';
+import get from 'lodash/get';
+import React from 'react';
+import { If, Then, Else } from 'react-if';
+import EventSchema from 'proptypes/schemas/EventSchema';
+import ConferenceRoomSchema from 'proptypes/schemas/ConferenceRoomSchema';
+import { Grid, Row, Col, Jumbotron } from 'react-bootstrap';
+import { DATE_DISPLAY_FORMAT } from 'helpers/DateHelper';
+
+import Clock from 'components/shared/time/Clock';
+import CurrentEvent from './event/CurrentEvent';
+import NextEvent from './event/NextEvent';
+
+import './conference_room.scss';
+
+const ConferenceRoomContainer = ({
+  currentEvent,
+  nextEvent,
+  conferenceRoom,
+  onUpdate
+}) => (
+  <Grid className="conference-room-container">
+    <Row style={{ backgroundColor: conferenceRoom.color }} className="room-header">
+      <Col xs={6}>
+        <h1>{conferenceRoom.title}</h1>
+      </Col>
+      <Col xs={6}>
+        <h3 className="pull-right"><Clock format={DATE_DISPLAY_FORMAT} /></h3>
+      </Col>
+    </Row>
+    <If condition={!!currentEvent || !!nextEvent}>
+      <Then>
+        <Row>
+          <Col xs={12} sm={8}>
+            <CurrentEvent event={currentEvent}
+                          nextEventStart={moment(get(nextEvent, 'start.date_time'))}
+                          onCompleted={onUpdate} />
+          </Col>
+          <Col xs={12} sm={4}>
+            <NextEvent event={nextEvent} />
+          </Col>
+        </Row>
+      </Then>
+      <Else>
+        <Col xs={12}>
+          <Jumbotron>
+            <h3>There are no more events for today</h3>
```

Comment:
Well maybe, but it was used there because the component is reusable and it gets rendered with different labeled. This one wouldn't.